_sass/_imla-nama.scss

```diff
@@ -122,23 +130,25 @@ p {
 .toc-index {
     margin-bottom: 2.6rem;
     text-align: right;
-    counter-reset: toc-index-count toc-index-count-roman;
+    counter-reset: toc-index-count toc-index-count-abjad;
 
     li::before {
```

Comment:
Let's give it a width here too (say 1.5em) and also center align it, so that the overall alignment of the TOC items is improved. (This issue was also present with Latin numerals and Lalezar, but was less noticable. The ب in abjad counting makes it more prominent now.)

Reply:
Done in [f4475d2](https://github.com/urduweb/imla/pull/17/commits/f4475d256dacb57c2d1c21c6a3c35622b238f76a). Used 1em width as 1.5em felt a bit too far apart.